commit 232645939b9739109cfadd92755375dd775b4768
Author: csagan5 <32685696+csagan5@users.noreply.github.com>
Date:   Wed Oct 11 23:22:05 2017 +0200

    [PATCH 45/60] promo: disable Google promotion fetching
    From: Jan Engelhardt <jengelh@inai.de>
    Date: Sun, 9 Nov 2014 01:54:27 +0100
    
    Documentation is scarce, and sprinkled with misleading acronyms.
    NTP is not NTP, for example. FWIW:
    
    // A PromoResourceService fetches data from a web resource server to
    // be used to dynamically change the appearance of the New Tab Page.
    // For example, it has been used to fetch "tips" to be displayed on
    // the NTP, or to display promotional messages to certain groups of
    // Chrome users.
    
    Whatever it is that it downloads, deactivate the one that leads to
    Google.
    
    References: https://github.com/iridium-browser/iridium-browser/issues/33

diff --git a/chrome/browser/signin/signin_promo.cc b/chrome/browser/signin/signin_promo.cc
index 4dd2f65..01f0f66 100644
--- a/chrome/browser/signin/signin_promo.cc
+++ b/chrome/browser/signin/signin_promo.cc
@@ -84,6 +84,7 @@ namespace signin {
 
 bool ShouldShowPromoAtStartup(Profile* profile, bool is_new_profile) {
   DCHECK(profile);
+  return false;
 
   // Don't show if the profile is an incognito.
   if (profile->IsOffTheRecord())
